stacks-signer/src/runloop.rs

```diff
@@ -937,21 +1071,60 @@ impl<C: Coordinator> SignerRunLoop<Vec<OperationResult>, RunLoopCommand> for Run
     }
 }
 
+fn check_nack_threshold(
+    nack_messages_received: &HashMap<CoordinatorMetadata, HashSet<u32>>,
+    nack_threshold: &u32,
+) -> Option<CoordinatorMetadata> {
+    nack_messages_received
+        .iter()
+        .find_map(|(coordinator_metadata, signers)| {
+            if signers.len() >= *nack_threshold as usize {
+                Some(coordinator_metadata.clone())
+            } else {
+                None
+            }
+        })
+}
+
+fn wrap_packet_in_packet_metadata(
```

Comment:
IMO this function isn't necessary, just use `PacketMetadata::new`

Reply:
Sure, I can remove it. I added it to keep the .into() conversion pattern used before the same